### PR TITLE
Add RCM option to first non-zero keyframe 'Send to t0'

### PR DESCRIPTION
**Pull request by roperzh**
OK to merge.

Short review.

Summary of changes:

This PR is mainly composed of two changes:

- Implements ["Add RCM option to first non-zero keyframe 'Send to t0'"](https://app.asana.com/0/687739386153745/682255795316585) (via c8227c0 and 5a64e89)
- Fixes a bug that raised from testing this which was present in the app, I filled [a task](https://app.asana.com/0/687739386153745/700316634325328) with steps to repro

Regressions to look for:

- Maybe updating keyframes

Completed checkin tasks:

- [x] Did manual testing of interrelated functionality
- [ ] Added measurement instrumentation (Mixpanel, etc.)
- [x] Wrote an automated test covering new functionality

### CODE REVIEW

**Comment by tsortwehttam** [comment]
Code LGTM, merging into `rc-3.5.0`

**Comment by tsortwehttam** [comment]
And thank you for the test 🙏

File: packages/haiku-timeline/src/components/Timeline.js [hunk 1]
```diff
@@ -521,6 +522,17 @@ class Timeline extends React.Component {
 
     items.push({ type: 'separator' })
 
+    items.push({
+      label: 'Send to t0',
```
Comment (stristr) [inline packages/haiku-timeline/src/components/Timeline.js:526]: Drive-by: is this the best name for this RCM option? (cc: @taylorpoe)
